### ISSUE TITLE
module 'torch.distributed' has no attribute 'optim'

### ISSUE THREAD

**Issue by jiang719** [pytorch/pytorch#65428]
## ❓ Questions and Help

I got this error: module 'torch.distributed' has no attribute 'optim'
Version:
pytorch==1.9.1
python==3.9.7

**Comment by SciPioneer** [comment]
Are you using Windows? I remember that `optim` does not work on WIndows for a while, but it then got fixed.

**Comment by jiang719** [comment]
> Are you using Windows? I remember that `optim` does not work on WIndows for a while, but it then got fixed.

No, I'm on Ubuntu 18.04.5 LTS. The interesting thing is this error is fixed when I install pytorch with the same command one more time.

`conda install pytorch torchvision torchaudio cudatoolkit=10.2 -c pytorch`

**Comment by SciPioneer** [comment]
@mrshenli Do you have any idea?

**Comment by SciPioneer** [comment]
It is possible that USE_DISTRIBUTED is not enabled when you installed PyTorch from source.
